nym-vpn-app/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "nym-vpn-app",
   "private": true,
-  "version": "0.2.4-dev",
+  "version": "1.0.0-dev",
```

Comment:
This branch `main` is targetting the next release after the one in the release branch, so convention is that it has a newer version `>1.0.0`. So something like `1.1.0-dev` or `1.0.1-dev`

Reply:
make sense if `1.0.0` has been released, which is not the case